kitsune/products/views.py

```diff
@@ -23,8 +23,19 @@ def product_list(request):
 
 
 @check_simple_wiki_locale
-def product_landing(request, slug):
-    """The product landing page."""
+def product_landing(request, slug: str) -> HttpResponse:
```

Comment:
Nice! Let's add also a type annotation for the request too, `request: HttpRequest`

Reply:
Done.